xpra/server/auth/keycloak.py

```diff
@@ -27,6 +30,9 @@ def __init__(self, **kwargs):
         self.client_id = kwargs.pop("client_id", KEYCLOAK_CLIENT_ID)
         self.client_secret_key = kwargs.pop("client_secret_key", KEYCLOAK_CLIENT_SECRET_KEY)
         self.redirect_uri = kwargs.pop("redirect_uri", KEYCLOAK_REDIRECT_URI)
+        self.claim_field = kwargs.pop("claim_field", KEYCLOAK_CLAIM_FIELD)
+        self.auth_groups = kwargs.pop("auth_groups", KEYCLOAK_AUTH_GROUPS).split()
```

Comment:
It could be worth doing: `self.auth_groups = set(...)` here instead of later. No risk either as `split()` always returns a list.